man/man8/zstream.8

```diff
@@ -149,6 +153,19 @@ Therefore, a deduplicated send stream can be received by running:
 Verbose.
 Print summary of converted records.
 .El
+.Nm
+.Cm recompress
+.Op Fl l Ar level
+.Ar algorithm
+.Xc
+Recompresses a send stream, provided on standard input, using the provided
+algorithm and optional level, and writes the modified stream to standard output.
+All WRITE records in the send stream will be recompressed, unless they fail
+to result in size reduction compared to being left uncompressed.
+The provided algorithm can be any valid value to the
+.Nm compress
+property.
```

Comment:
I think you could do `zfs set compression=gzip-3` (and similar for zstd), but it seems like you have to specify it here as `recompress -l 3 gzip`.  It would be nice if it did take the same values as the property, but if that's too big of a pain, let's change the documentation here.

Reply:
According to the manpage, `zstd-N` is how you specify the level.  So is `-l` only used for lz4, which doesn't have any way to specify the level via the property?
```
     compression=on|off|gzip|gzip-N|lz4|lzjb|zle|zstd|zstd-N|zstd-fast|zstd-fast
-N

...
       The zstd compression algorithm provides both high compression ratios
       and good performance.  You can specify the zstd level by using the
       value zstd-N, where N is an integer from 1 (fastest) to 19 (best
       compression ratio).  zstd is equivalent to zstd-3.
```